### ISSUE TITLE
Every time Obsidian is restarted, the hidden effects that have been pre-set will be invalidated

### ISSUE THREAD

**Issue by DanShui-Yu** [kelszo/obsidian-file-explorer-plus#10]
### What happened?

The hidden effects will only take effect after closing and reopening the plugin. I have tested that this bug is unrelated to whether the Custom File Explorer Sorting plugin is enabled or not.

### What did you expect to happen?

The hidden effects are automatically applied when Obsidian starts up.

### Can you reproduce the bug, if so how? As detailed but minimal as possible.

yes

### Plugin version

1.1.4

### Obsidian version

1.5.8

### OS

Windows

**Comment by kelszo** [comment]
Can you please specify more, it works for me

**Comment by kelszo** [comment]
I can’t replicate this, but to me it seems like a race condition between plugins. What plugins do you have installed?

**Comment by DanShui-Yu** [comment]
![image](https://github.com/kelszo/obsidian-file-explorer-plus/assets/69069739/eb2a1c7c-3149-4223-a6bd-6cb681be3af0)

**Comment by kelszo** [comment]
This is, with 99% chance, due to a conflict with one of the other plugins. 

I would try to see if only having this plugin activated fixes it. You can try in a completely different vault.

**Comment by kelszo** [comment]
Closing due to no update. Can't replicate. If you succeed in replicating this in an empty workspace please re-open

**Comment by DanShui-Yu** [comment]
I used the binary search method to troubleshoot, shutting down all plugins and then turning them back on in batches, and the bug did not occur during the process. I also tried to disable all plugins en masse and then re-enable them, but the bug still did not appear. However, when I kept other plugins enabled and restarted File Explorer++ alone and then restarted Obsidian, the bug reappeared; restarting File Explorer++ and then one or two other plugins, the bug still occurred, but it was resolved again when I restarted all plugins.
